### ISSUE TITLE
Determine height on longest object

### ISSUE THREAD

**Issue by kaspar-allenbach** [malsup/cycle2#33]
Hi.

In my Cycle2 Slideshow, the second slide is the longest one. So because cycle 2 determines the height based on the first slide, the second one gets cut off.

Is it possible to determine the height based on the longest slide, no matter if its first, second or what ever?

Thank you.

**Comment by malsup** [comment]
Use:

```
data-cycle-auto-height="calc"
```

to tell C2 to calculate sentinel slide.

http://jquery.malsup.com/cycle2/api/#auto-height

**Comment by kaspar-allenbach** [comment]
Thank you :)

**Comment by kaspar-allenbach** [comment]
Hmm. With this tag C2 has no height at all. Nothing is displayed anymore. Is this becaus I have `data-cycle-slides="> div"` and no Images?

**Comment by malsup** [comment]
Can you provide a link?  I'd like to debug that; sounds pretty odd.

**Comment by kaspar-allenbach** [comment]
Here: http://dekonstruktiv.ch/sonja_muehlemann/static/
Right beside the television is a sync Slideshow with the text to the image slides. Thats the place where it should retreive the calc height.

**Comment by malsup** [comment]
Oh, I guess I forgot to mention that you need to update C2.  I just added that 'calc' feature yesterday.  Sorry!  :-)

**Comment by kaspar-allenbach** [comment]
Oh. Yeah that helped quite a lot ;)

Thanks.

**Comment by marszalek** [comment]
Hi,

I've got the same problem, autoHeight: 'calc', but sentinel is always the same - first slide.
I got 2 sliders, its tallest slides are the third ones.
I've downloaded the development version of plugin and sent to console the height's of each slides, calculated in calcSentinelIndex function, but it returns 0 for all the slides.
i am initializing slider after DOM loads (the same problem is on document.ready).

Any help would be much appreciated.

Link to the site: http://navajo.neconserver.pl/services/customer-references-pr/ (mobile version -> less than 990px width)
